src/helpers/sortByHelpers.js

```diff
@@ -0,0 +1,19 @@
+const sortByLabelAndName = (array) => {
+  const sortFunc = (a, b) => {
+    let x, y
+    if (a.data.label !== undefined || !b.data.label !== undefined) {
+      x = a.data.label
+      y = b.data.label
+    } else {
+      x = a.name
+      y = a.name.toLowerCase()
+    }
+
+    return x.toLowerCase().localeCompare(y.toLowerCase())
```

Comment:
If a folder/task has no label I get this error.
![image](https://github.com/user-attachments/assets/420efefe-d42d-4063-a4e7-06fdc02f2721)

Reply:
Maybe this file could be written in typescript, would that have prevented this?